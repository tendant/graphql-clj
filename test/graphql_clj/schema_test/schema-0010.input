enum LengthUnit {
    METER
    FOOT
    INCH
    CENTIMETER
 }

 type Starship {
   id: ID!
   name: String!
   length(unit: LengthUnit = METER): Float
 }