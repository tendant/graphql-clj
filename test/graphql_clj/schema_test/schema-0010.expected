{:enums {:LengthUnit {:values [:METER :FOOT :INCH :CENTIMETER]}},
 :objects
 {:Starship
  {:fields
   {:id {:type (non-null ID)},
    :name {:type (non-null String)},
    :length {:args {"unit" {:type LengthUnit}}, :type Float}}}}}
